src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -5929,3 +5930,25 @@ def test_topic_migration_to_an_account():
             get_config_cluster(),
         )
         admin(["account", "rm", "--account-id", account_id], get_config_cluster())
+
+def test_persistent_notification_shard_config_change(conn): 
+    """ test persistent notification shard config change """
+    """ test to check if notifications work when config value for determining num_shards is changed..."""
+    default_config_value = 11
+    config_values = [0, 1, 3, 97, 111]
+    for config_value in config_values:
+        change_rgw_config_option(get_config_port(), 'rgw_bucket_persistent_notif_num_shards', config_value)
+        if conn == 'http':
+            test_notification_push_http()
+        elif conn == 'kafka':
+            test_notification_push_kafka()
+    ## changing to default value for other tests
+    change_rgw_config_option(get_config_port(), 'rgw_bucket_persistent_notif_num_shards', default_config_value)
+
+@attr('http_test')
```

Comment:
please change the attribute to "manual_test" so they won't run in teuthology.
the `change_rgw_config_option` function won't work there